tools/releases/sdf-release.yml

```diff
@@ -0,0 +1,80 @@
+trigger: none
+resources:
+  repositories:
+    - repository: GovernedTemplates
+      type: git
+      name: OneBranch.Pipelines/GovernedTemplates
+      ref: refs/heads/main
+  pipelines:
+    - pipeline: _OfficeDev.microsoft-teams-library-js
+      source: OfficeDev.microsoft-teams-library-js
+      project: ISS
+variables:
+  - name: System.Debug
+    value: true
+extends:
+  template: v2/Microsoft.Official.yml@GovernedTemplates
+  parameters:
+    platform:
+      name: m365
+      workload: IC3
```

Comment:
>workload: IC3 [](http://example.com/codeflow?start=6&length=13)

I'm going to assume this is correct even though we aren't part of IC3?

Reply:
I can ask the support team for confirmation. This was generated by the OneBranch tooling, but the allowed options are Substrate, SPO, OfficeOnline, IC3, Teams, Outlook, and Yammer. None of which seem like an obvious choice for our releases.

Reply:
Asked and answered. We should use Substrate as a placeholder for now, and this value will be optional in the future